src/Repository/EventRegistrationRepository.php

```diff
@@ -188,11 +188,44 @@ public function countEventParticipantsInReferentManagedArea(Adherent $referent,
             ->groupBy('yearmonth')
         ;
 
-        $query = RepositoryUtils::addStatstFilter($filter, $query);
+        if ($filter) {
+            $query = RepositoryUtils::addStatstFilter($filter, $query);
+        }
 
         return RepositoryUtils::aggregateCountByMonth($query->getQuery()->getArrayResult(), 'event_participants');
     }
 
+    public function countEventParticipantsInReferentManagedAreaInAtLeastOneCommittee(Adherent $referent, StatisticsParametersFilter $filter = null, int $months = 5): array
+    {
+        $this->checkReferent($referent);
+
+        $query = $this->createQueryBuilder('event_registrations')
+            ->select('DISTINCT event_registrations.emailAddress, COUNT(DISTINCT event_registrations) AS count, YEAR_MONTH(event.beginAt) as yearmonth')
+            ->join(Event::class, 'event', Join::WITH, 'event_registrations.event = event.id')
```

Comment:
`->join('event_registrations.event', 'event')`, no?

Reply:
yes thanks !

Reply:
In fact no, it's because I only want to join on `Event` and not `BaseEvent` (Basically I don't want citizenAction)